.github/CODEOWNERS

```diff
@@ -82,66 +82,15 @@
-/templates/ts/non-sso-tab @hund030 @eriolchan @huimiu
-/templates/csharp/non-sso-tab @JerryYangKai @eriolchan @hund030 @huimiu
-/templates/csharp/sso-tab @JerryYangKai @eriolchan @hund030 @huimiu
-/templates/js/default-bot @JerryYangKai @eriolchan @Siglud @Yukun-dong
-/templates/ts/default-bot @JerryYangKai @eriolchan @Siglud @Yukun-dong
-/templates/js/default-bot-message-extension @JerryYangKai @eriolchan @Siglud @Yukun-dong
-/templates/ts/default-bot-message-extension @JerryYangKai @eriolchan @Siglud @Yukun-dong
-/templates/js/message-extension @JerryYangKai @eriolchan @Siglud @Yukun-dong
-/templates/ts/message-extension @JerryYangKai @eriolchan @Siglud @Yukun-dong
-/templates/js/link-unfurling @JerryYangKai @eriolchan @Siglud @Yukun-dong
-/templates/ts/link-unfurling @JerryYangKai @eriolchan @Siglud @Yukun-dong
-/templates/csharp/message-extension @JerryYangKai @eriolchan @Siglud @Yukun-dong
-/templates/csharp/link-unfurling @JerryYangKai @eriolchan @Siglud @Yukun-dong
-/packages/fx-core/src/common/template-utils @hund030 @huimiu @eriolchan
-/packages/fx-core/tests/common/templateFetch.test.ts @hund030 @huimiu @eriolchan
-
-/templates/js/command-and-response @kimizhu @dooriya @swatDong
-/templates/js/notification-http-timer-trigger @kimizhu @dooriya @swatDong
-/templates/js/notification-http-trigger @kimizhu @dooriya @swatDong
-/templates/js/notification-restify @kimizhu @dooriya @swatDong
-/templates/js/notification-timer-trigger @kimizhu @dooriya @swatDong
-/templates/js/workflow @kimizhu @dooriya @swatDong
-/templates/js/m365-message-extension @kimizhu @swatDong @kuojianlu
-/templates/js/m365-tab @kimizhu @swatDong @kuojianlu
-/templates/ts/command-and-response @kimizhu @dooriya @swatDong
-/templates/ts/notification-http-timer-trigger @kimizhu @dooriya @swatDong
-/templates/ts/notification-http-trigger @kimizhu @dooriya @swatDong
-/templates/ts/notification-restify @kimizhu @dooriya @swatDong
-/templates/ts/notification-timer-trigger @kimizhu @dooriya @swatDong
-/templates/ts/workflow @kimizhu @dooriya @swatDong
-/templates/ts/m365-message-extension @kimizhu @swatDong @kuojianlu
-/templates/ts/m365-tab @kimizhu @swatDong @kuojianlu
-/templates/csharp/command-and-response @kimizhu @dooriya @swatDong
-/templates/csharp/notification-http-timer-trigger @kimizhu @dooriya @swatDong
-/templates/csharp/notification-http-trigger @kimizhu @dooriya @swatDong
-/templates/csharp/notification-webapi @kimizhu @dooriya @swatDong
-/templates/csharp/notification-timer-trigger @kimizhu @dooriya @swatDong
-/templates/csharp/workflow @kimizhu @dooriya @swatDong
-
 /packages/fx-core/src/common/local @kimizhu @swatDong @kuojianlu @XiaofuHuang
 /packages/fx-core/tests/common/local @kimizhu @swatDong @kuojianlu @XiaofuHuang
 /packages/fx-core/src/common/m365 @kimizhu @swatDong @kuojianlu
 /packages/fx-core/tests/common/m365 @kimizhu @swatDong @kuojianlu
+/packages/fx-core/src/component/resource/botService @kimizhu @swatDong @kuojianlu
```

Comment:
There's one `botAadApp.ts` remaining in this folder but is used for `botAadApp/create` action only. Any plan to move this file?

Reply:
Just noticed it's already done by https://github.com/OfficeDev/TeamsFx/pull/9047.